public/app/features/explore/Wrapper.test.tsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import Wrapper from './Wrapper';
+import { configureStore } from '../../store/configureStore';
+import { Provider } from 'react-redux';
+import { store } from '../../store/store';
+import { setDataSourceSrv } from '@grafana/runtime';
+import {
+  ArrayDataFrame,
+  DataQueryResponse,
+  DataSourceApi,
+  DataSourceInstanceSettings,
+  FieldType,
+  QueryEditorProps,
+  ScopedVars,
+} from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+
+import { setTimeSrv } from '../dashboard/services/TimeSrv';
+import { from, Observable } from 'rxjs';
+import { updateLocation } from '../../core/reducers/location';
+import Mock = jest.Mock;
```

Comment:
Thats interesting, I think this is some autocomplete thing from Goland (it adds imports automatically if it know what to import). Maybe because it is used as a type or something? Will have to check if this makes sense otherwise will just change it.

Reply:
ah ok, I did a quick search and can only see one instance of the same import in `language_provider.test.ts`, so I guess it works.

Reply:
So the problem is that Mock is a type so you cannot do `const {Mock} = jest;` and I assume it could be some typescript specific import thing. At the same time `type Mock = jest.Mock` seems more clear.